plugin/xmark.vim

```diff
@@ -38,6 +38,7 @@ let s:files = {
 \ 'xsize':  s:dir . '/ext/xsize'
 \ }
 let s:app = 'Google Chrome'
+let s:app_loc = '/Applications/Google Chrome.app'
```

Comment:
Also, agreed on the hard-coded path. I'm not overly familiar with applescript (or vimscript for that matter), so I'm not sure what the most reasonable approach is.